platform/windows/detect.py

```diff
@@ -452,15 +481,10 @@ def configure_msvc(env, vcvars_msvc_config):
         arch_subdir = "arm64" if env["arch"] == "arm64" else "x64"
 
         # PIX
-        if env["pix_path"] != "":
+        if env["pix_path"] != "" and os.path.exists(env["pix_path"]):
```

Comment:
Also, I do not think we should enable PIX by default (or enable it only for debug builds only), it's a debugging tool, and unnecessary for release templates.

Reply:
I could add changes to https://github.com/godotengine/godot/pull/87609, but it's maybe better if you do a follow-up directly to handle making PIX opt-in / debug only.

Reply:
I'll do a follow-up later.

Reply:
OK, Contact me if there is anything that needs me to do🫡